priv/repo/migrations/20181005115709_update-student-session-constraint.exs

```diff
@@ -1,4 +1,4 @@
-defmodule :"Elixir.Nexpo.Repo.Migrations.Update-student-session-constraint" do
+defmodule Nexpo.Repo.Migrations.UpdateStudentSessionConstraint do
```

Comment:
Shouldn't it look like this?

Reply:
Dont think so https://github.com/careerfairsystems/nexpo/blob/master/priv/repo/migrations/20181001082745_update-student-session-on-delete.exs

Reply:
Or that one is wrong also

Reply:
Lul that looks weird xD

Reply:
Can't change the past tho